rpm/sailfish-connman-plugin-suspend-wmtwifi.spec

```diff
@@ -17,22 +17,22 @@ BuildRequires: pkgconfig(libnl-genl-3.0)
 
 %description
 This is a package which makes sure devices with a gen2 /dev/wmtWifi driver can
-sucessfully suspend while WLAN is active.
+successfully suspend while WLAN is active.
 
 %prep
 %setup -q -n %{name}-%{version}
 
 %build
-make %{_smp_mflags} KEEP_SYMBOLS=1 release
+make %{_smp_mflags} KEEP_SYMBOLS=1 WMTWIFI_SUPPORT=1 release
 
 %install
 rm -rf %{buildroot}
 mkdir -p %{buildroot}/%{plugin_dir}
 %make_install
-mkdir -p %{buildroot}/etc/udev/rules.d
-install -m 664 %{SOURCE1} %{buildroot}/etc/udev/rules.d/70-wifi-powersave.rules
+mkdir -p %{buildroot}/lib/udev/rules.d
+install -m 664 %{SOURCE1} %{buildroot}/lib/udev/rules.d/70-wifi-powersave.rules
 
 %files
 %defattr(-,root,root,-)
 %{plugin_dir}/*.so
-%{_sysconfdir}/udev/rules.d/70-wifi-powersave.rules
+/lib/udev/rules.d/70-wifi-powersave.rules
```

Comment:
This and above related changes would look nicer in a separate commit

Reply:
Changes moved to separate commit.